docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,483 @@
+   await srv.serve()
+
+if __name__ == "__main__":
+   asyncio.run(main())
+```
+
+You can modify this example code as necessary.
+
+Now, go back to your robot's page on [the Viam app](https://app.viam.com) and navigate to the **CONFIG** tab, then to the **REMOTES** sub-tab.
+
+Add your sensor server as a remote part, called `my-sensor-server`:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+<img src="../../img/plant-watering-pi/sensor-remote-config-builder.png" alt="Creation of a custom sensor remote in the Viam app config builder." style="max-width:800px" />
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+// "components": [ board & motor ... ],
+"remotes": [
+    {
+      "name": "my-sensor-server",
+      "address": "localhost:9090"
+    }
+]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+<br>
+
+Then, navigate to the **PROCESSES** subtab and create a process called `run-sensor-server`:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+<img src="../../img/plant-watering-pi/sensor-process-config-builder.png" alt="Creation of a pump motor in the Viam app config builder." style="max-width:800px" />
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  // "components": [ board & motor ... ] ,
+  // "remotes" [ my-sensor-server ... ] ,
+  "processes": [
+    {
+      "id": "run-sensor-server",
+      "log": true,
+      "name": "sudo",
+      "args": [
+        "-u",
+        "<my_username>",
+        "python",
+        "sensor.py"
+      ],
+      "cwd": "/home/<my_username>/plant-watering-robot"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Add Python Control Code
+
+Follow these instructions to start working on your Python Control Code:
+
+1. Navigate to your robot's page in [the Viam app](https://app.viam.com), and click on the **CODE SAMPLE** tab.
+Follow the instructions in this tab.
+2. Click **COPY CODE** to copy a code sample that establishes a connection with your robot when run.
+3. Paste this code sample into a new file in your `plant-watering-robot` directory.
+4. Name the file <file>plant-watering-robot.py</file>, and save it.
+  
+For example, run the following commands to create and open the file:
+
+``` shell
+cd plant-watering-robot
+touch plant-watering-robot.py
+vim plant-watering-robot.py
+```
+
+Now, you can add code into <file>plant-watering-robot.py</file> to write the logic that defines your plant watering system.
+
+<img src="../../img/plant-watering-pi/main-py.png" alt="The main.py function from your robot's Code Sample tab, ready to edit." style="max-width:800px" />
+
+<br>
+</br>
+
+Use the Viam [motor](/components/motor#api) and [sensor](/components/sensor#control-your-sensor-with-viams-client-sdk-libraries) API methods.
+Determine at what [analog value from the Soil Moisture readings](#test-your-soil-moisture-readings-on-your-pi) you want to water your plant.
```

Comment:
can you give an example? Or a reasonable start value?

Reply:
or tell them to use the value from earlier testing

Reply:
lmk how change looks haha